packages/terra-clinical-slide-group/src/SlideGroup.jsx

```diff
@@ -0,0 +1,60 @@
+import React, { PropTypes } from 'react';
+import classNames from 'classnames';
+import CSSTransitionGroup from 'react-transition-group/CSSTransitionGroup';
+
+import Slide from './Slide';
+
+import './SlideGroup.scss';
+
+const propTypes = {
+  /**
+   * The array of components for the group. Only the last component is visible. The others are hidden but still mounted.
```

Comment:
I would be curious to see what others thought. I feel like we tend to handle the "weird logic"on our end instead of leaving that knowledge on the consumer.